app/assets/stylesheets/common/_markus.scss

```diff
@@ -1059,6 +1059,21 @@ nav {
   transform-origin: right top;
 }
 
+.rotate90 {

```

Comment:
I'm not sure why you resolved this issue, but the code here looks like an exact duplicate of lines 1047-1060. Might just be a merge error or something, but you should delete this.